src/controllers/requestController.js

```diff
@@ -52,6 +53,9 @@ const Request = {
         index += 1;
       }
       Req.trips = Trips;
+      await db.Users.update({ rememberMe }, {
```

Comment:
If the `rememberMe` `=/=` `user.rememberMe`, then update the user data. If the user's